.changes/mac-app-hide.md

```diff
@@ -0,0 +1,7 @@
+---
+"tauri-runtime-wry": minor
+"tauri-runtime": minor
+"tauri": minor
+---
+
+Add `AppHandle::hide()` for hiding the entire application on macOS
```

Comment:
what about `AppHandle::show()` and `AppHandle::hide_others` ? I believe you add them to tao, right ?

Reply:
Wasn't planning on adding `show` in this PR, but I'll go for it. They're all in tao already, yeah.

I don't really think it's worth adding `hide_others` though

Reply:
Add it while you're at it anyway and you never know if someone will need it.

Reply:
I could, but not sure I want the method to be there if it might never even be used, not even Electron has that one. Maybe that's a weird concern though?

Reply:
we don't need to be electron and we already expose it in TAO and this is just simply forwarding it so I don't see a reason why not.

Reply:
Just so there's not a bunch of unused methods to sift through to find what you want. I'm not mentioning Electron because Tauri needs to be like it. Electron has hundreds of thousands of users, so if Electron is missing a simple feature, it's a good indication that nobody wants it. Either way, it's not a big deal, can add it if you still think it should be.

Reply:
I don't have a strong opinion about it, do what you feel is right,

Reply:
Then I'll leave it out for now. Also, added `show`.